panoptes_client/project.py

```diff
@@ -127,5 +127,13 @@ def add_workflows(self, workflows):
             'workflows',
         )
 
+    @property
+    def avatar(self):
+        """
+        A dict containing metadata about the user's avatar.
+        """
+
+        return Project.http_get('{}/avatar'.format(self.id))[0]
```

Comment:
Is the `Avatar` class still needed, since it's not being used here? If we're keeping that class I think it'd be better to return it here.